Starwar/Assets/Scripts/Player Control/Player_Control.cs

```diff
@@ -48,14 +53,22 @@ private void FixedUpdate()
         if (Input.GetButton("Fire1"))
         {
             Debug.Log("Fire bullet");
+            soundController.playLaser();
             machineGun.Fire();
         }
 
         if (Input.GetButtonDown("Fire2"))
         {
-            Debug.Log("Fire Missile");
+            Debug.Log("Fire Missile");  
+            soundController.playMissile(); 
             missileLauncherManager.LockOn();
             missileLauncherManager.Fire();
         }
+
+        
+        soundController.playEngine(lastSpot - transform.position);
```

Comment:
play engine sound when "Vertical" axis > 0  (line 26) is more intuitive though, because the ship can turn off the engine and just slide using momentum.

Reply:
removed last position, added to the first if statement and added an if to turn the engine sound off